City-of-Helsinki/drupal-module-helfi-tpr
# UHF-8390 D10 deprecation

Updated deprecated drupal_get_path -function call
Entity query access check.

How to test:
composer require drupal/helfi_tpr:dev-UHF-8390
composer require drupal/upgrade_status and drush en -y upgrade_status
run drush upgrade_status:analyze helfi_azure_fs
You should only see deprecation errors from prophecy / prophesize method.

Run the update hooks which changed.
